tests/e2e/support/cta/files/sidebar.ts

```diff
@@ -37,10 +37,10 @@ export const openPanel = async ({
   const panelSelector = page.locator(`#sidebar-panel-${name}-item-select`)
   const nextPanel = page.locator(`#sidebar-panel-${name}-item`)
   if (await panelSelector.count()) {
-    await panelSelector.click()
     await Promise.all([
       locatorUtils.waitForEvent(nextPanel, 'focus'),
-      locatorUtils.waitForEvent(nextPanel, 'transitionend')
+      locatorUtils.waitForEvent(nextPanel, 'transitionend'),
+      panelSelector.click()
```

Comment:
is there a reason why the click event is part of the promise?

Reply:
I thought that was the reason for this error: https://drone.owncloud.com/owncloud/web/21580/10/18

Reply:
Ah thanks for explaining ❤️